optas/models.py

```diff
@@ -1818,12 +1818,153 @@ def get_global_link_axis_function(
     ) -> cs.Function:
         """! Get function for computing the link axis in the global frame.
 
-@param link Name of the end-effector link.
-@param axis The axis (direction vector) defined in the end-effector frame. If 'x', 'y', 'z' is passed then the corresponding sub-array of the homogenous transform is used.
-@param numpy_output When true, the output will be a NumPy array.
-@return A CasADi function that computes the link axis for a given joint state. If n > 1 then an array is computed whose columns correspond to the respective joint state in the input.
+        @param link Name of the end-effector link.
+        @param axis The axis (direction vector) defined in the end-effector frame. If 'x', 'y', 'z' is passed then the corresponding sub-array of the homogenous transform is used.
+        @param numpy_output When true, the output will be a NumPy array.
+        @return A CasADi function that computes the link axis for a given joint state. If n > 1 then an array is computed whose columns correspond to the respective joint state in the input.
         """
         get_global_link_axis = functools.partial(self.get_global_link_axis, axis=axis)
         return self.make_function(
             "a", link, get_global_link_axis, n=n, numpy_output=numpy_output
         )
+
+    def rnea(self, q: ArrayType, qd: ArrayType, qdd: ArrayType) -> ArrayType:
```

Comment:
Please add the decorator `@arrayify_args`:
```python
@arrayify_args
def rnea(...):
    ..
```
This ensures what ever the user passes into the function is in the correct format.